pandasai/__init__.py

```diff
@@ -485,9 +519,10 @@ def run_code(
         )
 
         environment: dict = {
+            "pd": pd,
             **{
-                alias: sys.modules[library]
-                for library, alias in ENVIRONMENT_DEFAULTS.items()
+                lib["alias"]: getattr(import_dependency(lib["module"]), lib["name"])
```

Comment:
When adding a `ast.Import` to the environment, we don't need to get any attributes from the module. Just the module itself.